### ISSUE TITLE
Importing strategies

### ISSUE THREAD

**Issue by yannickb85** [bupticybee/TexasSolver#167]
I'm presuming that saving strategies is in sort the way to save your simulations, am I wrong in thinking that? I wonder then how to import those saved strategies afterwards to review them later.

Otherwise is there any way to save the simulations that I compute in order to review them later or do I need to re-run them each time I open the program.

Thank you!

**Comment by bupticybee** [comment]
> I'm presuming that saving strategies is in sort the way to save your simulations, am I wrong in thinking that? I wonder then how to import those saved strategies afterwards to review them later.
> 
> Otherwise is there any way to save the simulations that I compute in order to review them later or do I need to re-run them each time I open the program.
> 
> Thank you!

Yes you are right, loading the saved strategy is still under development. Sorry it can't be done now.

**Comment by Robyn12** [comment]
I'd like to see also a feature to import multiple .json solutions. So that it's faster to run through multiple solutions. Maybe even solutions explorer as different executable. I think it would be great asset, if I decide to run multiple solutions with console_solver.
Thank you for this great software, I appreciate your work! :3

**Comment by bupticybee** [comment]
> I'd like to see also a feature to import multiple .json solutions. So that it's faster to run through multiple solutions. Maybe even solutions explorer as different executable. I think it would be great asset, if I decide to run multiple solutions with console_solver. Thank you for this great software, I appreciate your work! :3

Yeah it's just I havn't got the time to implement the feature recently. Happy if you want to try~

**Comment by Robyn12** [comment]
I don't have QT license, and no corporate-email, right now. Is it possible to get the license legally?

**Comment by bupticybee** [comment]
> I don't have QT license, and no corporate-email, right now. Is it possible to get the license legally?

Why don't you just use QT opensource version?

**Comment by Anguschen0430** [comment]
Is there any update on the load_strategy?

**Comment by Takumi-trbl** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
